ts/page-load.ts

```diff
@@ -0,0 +1,26 @@
+import generateChart from './data-display';
+
+const displayHeader = () => {
+    const content = document.querySelector('#content');
+    const header = document.createElement('div');
+    header.setAttribute('id', 'header-container');
+    const welcomeHeader = document.createElement('div');
+    welcomeHeader.id = 'welcome-header';
+    welcomeHeader.innerHTML = '<br>Power Data App<br>Welcome, here\'s your data!<br>';
+    header.append(welcomeHeader);
+    content.append(header);
```

Comment:
pls no. you're already using `innerHTML` anyway. just
```js
content.innerHTML = '<header><h1>Power Data App</h1></header>';
```
(try to avoid `div` and prefer semantic elements)